src/node_config.rs

```diff
@@ -191,6 +253,50 @@ pub fn cli_parse() -> CliConfig {
             .help("Disable LAN discovery"))
         .get_matches();
 
+    match matches.subcommand() {
+        ("config", Some(m)) => run_config(m),
+        _ => run_args(&matches),
+    }
+}
+
+/// Parse settings from a saved file.
+fn parse_config(config_path: String) -> NodeConfig {
+    let mut settings = Config::default();
+
+    settings.set_default("log-type", "Stderr").expect("Can't set default value for `log-type`");
+    settings.set_default("motd", "This is tox-rs").expect("Can't set default value for `motd`");
+    settings.set_default("no-lan", "False").expect("Can't set default value for `no-lan`");
+    settings.set_default("threads", "1").expect("Can't set default value for `threads`");
+
+    let config_file = if !Path::new(&config_path).exists() {
+        panic!("Can't find config file {}", config_path);
+    } else {
+        CfgFile::with_name(&config_path)
+    };
+
+    settings.merge(config_file).expect("Merging config file with default value fails");
+
+    // Collect unrecognized fields to warn about them
+    let mut unused = Set::new();
+    let mut config: NodeConfig = serde_ignored::deserialize(settings, |path| {
+        unused.insert(path.to_string());
+    }).expect("Can't deserialize config");
+
+    config.unused = unused;
+    config.sk_passed_as_arg = false;
+    config.lan_discovery_enabled = !config.lan_discovery_enabled;
```

Comment:
I'd suggest to rename `no-lan` field in the config file to `lan-discovery`. `no-lan` was used as CLI flag to disable enabled by default lan discovery. But in the config we set it as boolean field. So it would be more straightforward and we wouldn't need this inversion.

Reply:
We'd better use the same keys in config and in args. If you suggest to rename `no-lan` to `lan-discovery` it should be done in args too, which is another PR